### DISCUSSION TITLE
Welcome to CFEngine Discussions!

### DISCUSSION THREAD

**Discussion by olehermanse** [cfengine/core#4435] [General]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about your experience with CFEngine.

**Comment by olehermanse** [comment]
I'll start this off:

I'm Ole Herman. I've been a CFEngine core developer since 2016, employed by [Northern.tech](https://northern.tech/). I am now the Product Manager for CFEngine. Interested to hear everyone's CFEngine stories and receive any feedback you might have. 🚀

🐦 [@olehermanse](https://twitter.com/olehermanse)

**Comment by larsewi** [comment]
Hi, my name is Lars Erik. I'm an intern at [Northern.tech](https://northern.tech/). My main focus is learning wizardry from the wizards of the CF Engine team. I'm also interested to hear everyone's CFEngine stories!

**Comment by craigcomstock** [comment]
I am Craig Comstock. My chosen title is "digger" as I like to dig into the details of thorny issues and poke around. I work on various parts of CFEngine: core, masterfiles and enterprise and just for fun try to use CFEngine for as much as I possibly can in my day-to-day computer interactions.

Eventually I'd like to not touch a computer hardly at all knowing that my CFEngine policy is keeping everything proper. :+1: 

Here is a blog post about my start into a "personal" policy which manages things around my own computer systems.

https://cfengine.com/company/blog-detail/personal-policy/

My passion is to make a very open and fun mobile phone device. Towards that end I am making a console-only UI for phones running [postmarketOS](http://postmarketos.org/) called shelli which uses a gesture recognition algorithm I came up with: https://gitlab.com/unrznbl/gesture/-/wikis/gesture---handwriting-for-linux-touchscreens

:elephant: [unrznbl](https://bsd.network/web/accounts/81048)
:bird: [_craig_comstock](https://twitter.com/craig_comstock_)

**Comment by amousset** [comment]
Hi :wave: !

I am Alexis Mousset, I work as developer on [RUDDER](https://rudder.io) (which uses CFEngine as configuration engine for Unix systems). I'm been using CFEngine quite intensively since 2015, worked on the CFEngine framework we use in RUDDER and made some contributions to CFEngine core along the way.

(I have a lot of CFEngine stories, don't know where to start :smile: )

**Comment by basvandervlies** [comment]
I am Bas van der Vlies working at the High Performance Computing department at [SURF](https://www.surf.nl/en) (formerly known  as SARA or SURFsara). I use CFEngine since 1997.  I have done al lot of contributions to the different parts of CFEngine.  The biggest was n 2013 when I was hired by the company for a week to improve the[ syntax checker](https://basvandervlies.blogspot.com/2012/11/a-better-cfengine-syntax-checker.html) (parser). 

We at  SURF have developed a [mustache/json services framework](https://github.com/basvandervlies/cf_surfsara_lib) that can be added to masterfiles or be used in your own framework. This is one of the many stories I have  😄!

**Comment by vpodzime** [comment]
Hi! :beers: :czech_republic: 
I'm Vratislav Podzimek and I have been working as a Swiss Army knife in the CFEngine team since 2018. I mostly focus on the C code base of the project, but I often poke into packaging, CI, platform enablement and many other things. Outside of the CFEngine world I'm a big FOSS enthusiast, a long time GNU/Linux user and a former main developer of the [libblockdev](https://github.com/storaged-project/libblockdev) library and some related projects.

:bird: [v_podzimek](https://twitter.com/v_podzimek)
https://cfengine.com/author/a10068/
https://storageapis.wordpress.com/author/vpodzime/

**Comment by tzz** [comment]
Hi! I'm Ted Zlatanov ex-CFEngine team and hope to be helpful and contribute to this community.

**Comment by zzamboni** [comment]
Hi everyone - I'm Diego Zamboni, another ex-CFEnginer, and author of the book [Learning CFEngine](https://cf-learn.info/). Happy to be here!

**Comment by nickanderson** [comment]
Hai everybody!

I'm [Nick Anderson](https://www.linkedin.com/in/hithisisnick/) ([@nickanderson](https://github.com/nickanderson)). I have been using CFEngine since 2010 after attending [Configuration Management Solutions with CFEngine 3](https://static.usenix.org/events/lisa10/training/tutonefile.html#t3)  by Mark Burgess ([@markburgess](https://github.com/markburgess/)) at LISA10 (~CFEngine version 3.1). I have been hacking at CFEngine ever since. I was honored to be named a [CFEngine Champion](https://cfengine.com/engage/cfengine-champions/) in 2012 and ended up joining [CFEngine AS](https://cfengine.com/engage/cfengine-champions/) (Now [Northern.tech](https://northern.tech)) in 2013 (nearly 8 years ago now).

I get involved in nearly every aspect of CFEngine, but I spend most of my time with users and policy. I frequent the [help-cfengine list/forum](https://groups.google.com/forum/#!forum/help-cfengine), [#cfengine on irc.freenode.net](https://webchat.freenode.net/#cfengine) | [gitter.im/cfenigne/core](https://gitter.im/cfengine/core).

I am an [org-mode](https://orgmode.org/) fanatic and authored [ob-cfengine3](https://github.com/nickanderson/ob-cfengine3) which makes working with CFEngine policy in org-mode a breeze.

If you have a question, I probably have an answer, but I don't promise you will like it.

:bird: [@cmdln_](https://twitter.com/cmdln_)
https://cfengine.com/author/a10042/
https://cmdln.org

**Comment by atsaloli** [comment]
Hello!  I am Aleksey Tsalolikhin and I am a CFEngine implementation consultant and trainer at [Vertical Sysadmin](http://www.verticalsysadmin.com). 

I've been using CFEngine since version 2.  When version 3 came out in 2008, I tried to read the documentation and I couldn't understand it.  So I took Mark's CFEngine class at the USENIX LISA conference.  I sat in the front row and pestered him with question after question until about halfway through the training everything just clicked and fell in and made sense, and aligned into an elegant tower of understanding.   

Since then, I've helped quite a few other sysadmins get started with CFEngine.  I really enjoy bringing about that "lightbulb" moment in others.

My training materials are available online at www.cfenginetutorial.org.

I had contributed an appendix (condensed language reference) to Diego's "Learning CFEngine 3" book.

Recognized as [CFEngine Community Champion](https://cfengine.com/engage/cfengine-champions/) in 2011.

Delighted to be part of this extraordinary community.  :)

**Comment by cody-valle** [comment]
Hi Everybody - My name is Cody Valle and I'm the Director of Community at CFEngine. I joined the team in 2019 and my core responsibility is working with new & existing users to better understand their use cases and how we can improve CFEngine to best fit their needs and requirements. It's great to meet all of you and I look forward being a part of the community!
